src/dev/integration/index.md

```diff
@@ -1,73 +1,210 @@
-
-The <GitHubRepo id="MerginMaps/python-api-client" /> repository contains the source code of the Python client module and more information on how to use it.
+## Installation
+The Python client module is the easiest way to programmatically use <MainPlatformNameLink />. You can use Python API or a command-line tool to easily work with your projects, such as to get project status, push and pull changes. You can also create user accounts and manage their roles.
 
 Python client is available in the PyPI repository and can be installed with `pip`:
 
 ```
 pip3 install mergin-client
 ```
 
-### Python module 
+## Command line interface
+For those who prefer using terminal, there is `mergin` command line tool shipped with the Python client. With several built-in commands, it is possible to download <MainPlatformName /> projects, push/pull changes, create or delete projects and more.
+
+For example, to download a <MainPlatformName /> project to a local folder:
+```
+mergin download john/project1 ~/mergin/project1
+```
+For more details, visit <GitHubRepo id="MerginMaps/python-api-client" />.
+
+## Python module 
 To use <MainPlatformNameLink /> from Python, just create `MerginClient` object and then use it. Here, for instance, to download a project:
 
 ```python
 import mergin
 
-client = mergin.MerginClient(login='john', password='topsecret')
+client = mergin.MerginClient(login='john', password='T0p_secret')
 client.download_project('lutraconsulting/Basic survey', '/tmp/basic-survey')
 ```
 If you have <QGISPluginName /> installed and you want to use it from the QGIS' Python console
 
 ```python
 import Mergin.mergin as mergin
-client = mergin.MerginClient(login='john', password='topsecret')
+client = mergin.MerginClient(login='john', password='T0p_secret')
 ```
 
-### Python command line interface
-For those who prefer using terminal, there is `mergin` command line tool shipped with the Python client. With several built-in commands, it is possible to download <MainPlatformName /> projects, push/pull changes, create or delete projects and more.
+## API reference - users
 
-For example, to download a <MainPlatformName /> project to a local folder:
+You can create new users and manage their roles using the following methods.
+
+::: warning API availability
+The following methods are available for Python API Client versions `0.10.0` or higher, using server versions `2025.2.0` or higher.
+:::
+
+### Create a user
+
+```python
+client.create_user(<email>, <password>, <workspace_id>, <workspace_role>, [username], [notify_user])
 ```
-mergin download john/project1 ~/mergin/project1
+
+The caller must be a workspace admin, owner or server administrator.
+
+Arguments:
+
+`email` (string): Must be a unique email.
+
+`password` (string): Must meet security requirements.
+
+`workspace_id` (int)⭐️ : The workspace ID where the user will be added.
```

Comment:
```suggestion
`workspace_id` (int) :large_blue_diamond: : The workspace ID where the user will be added.
```

I think we use ⭐️ elsewhere as as symbol for required variables. To keep it consistent, maybe another symbol would work better here.

Reply:
Good point, thanks

Reply:
Used :information_source: icon and added also hover text.